Emby.Server.Implementations/Library/UserDataManager.cs

```diff
@@ -220,7 +220,7 @@ public bool UpdatePlayState(BaseItem item, UserItemData data, long? reportedPosi
             var hasRuntime = runtimeTicks > 0;
 
             // If a position has been reported, and if we know the duration
-            if (positionTicks > 0 && hasRuntime && !(item is AudioBook))
+            if (positionTicks > 0 && hasRuntime && !(item is AudioBook) && !(item is Book))
```

Comment:
```suggestion
            if (positionTicks > 0 && hasRuntime && item is not AudioBook && item is not Book)
```

Reply:
👍🏻 Shall use this suggestion for readability :)